features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Edit.jsx

```diff
@@ -0,0 +1,305 @@
+            </Button>
+            <Dialog
+                open={open}
+                disableBackdropClick
+                disableEscapeKeyDown
+                maxWidth='sm'
+                fullWidth
+                aria-labelledby='confirmation-dialog-title'
+            >
+                <DialogTitle id='confirmation-dialog-title'>
+                    <FormattedMessage
+                        id='ServiceCatalog.Listing.Edit.service'
+                        defaultMessage='Edit Service'
+                    />
+                </DialogTitle>
+                <DialogContent>
+                    <TextField
+                        name='name'
+                        label={(
+                            <>
+                                <FormattedMessage
+                                    id='ServiceCatalog.Listing.Edit.service.name.label'
+                                    defaultMessage='Service'
+                                />
+                                <sup className={classes.mandatoryStar}>*</sup>
+                            </>
+                        )}
+                        value={name}
+                        variant='outlined'
+                        fullWidth
+                        helperText={(
+                            <FormattedMessage
+                                id='ServiceCatalog.Listing.Edit.service.name.helper.text'
+                                defaultMessage='Name of the service'
+                            />
+                        )}
+                        onChange={handleChange}
+                    />
+                </DialogContent>
+                <DialogContent>
+                    <TextField
+                        name='serviceUrl'
+                        label={(
+                            <>
+                                <FormattedMessage
+                                    id='ServiceCatalog.Listing.Edit.service.url.label'
+                                    defaultMessage='Service URL'
+                                />
+                                <sup className={classes.mandatoryStar}>*</sup>
+                            </>
+                        )}
+                        value={serviceUrl}
+                        fullWidth
+                        variant='outlined'
+                        helperText={(
+                            <FormattedMessage
+                                id='ServiceCatalog.Listing.Edit.service.url.text'
+                                defaultMessage='URL of the Service'
+                            />
+                        )}
+                        onChange={handleChange}
+                    />
+                </DialogContent>
+                <Grid container spacing={0}>
+                    <Grid item md={6}>
+                        <DialogContent>
+                            <TextField
+                                name='definitionType'
+                                select
```

Comment:
![image](https://user-images.githubusercontent.com/3313885/100128215-051a7180-2ea6-11eb-9de4-427abbae7165.png)

Shall we use selector positioning to avoid misplaced dropdown positioning 

example

```javascript
SelectProps={{
                    MenuProps: {
                        anchorOrigin: {
                            vertical: "bottom",
                            horizontal: "left"
                        },
                        getContentAnchorEl: null
                    }
                }}
```